Tests/FoundationTests/URLExtensionsTests.swift

```diff
@@ -33,5 +33,17 @@ final class URLExtensionsTests: XCTestCase {
 		url.appendQueryParameters(params)
 		XCTAssertEqual(url, queryUrl)
 	}
+    
+    func testQueryItemsVar() {
```

Comment:
Can you name this `testQueryParameters`? We use the naming scheme of `test` + method or property name.